plugins/woocommerce/changelog/52610-update-offline-payment-gateway

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Updated Offline Payments Settings layout to match the design
```

Comment:
we may omit this since we'll add one changelog item for the functionality, see p1730810180012819-slack-C03KTTK2YMA

Reply:
Got it, thanks! Removing it now.